src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -355,6 +355,43 @@ public StudyController(StudyService studyService) {
         return ResponseEntity.ok().body(studyService.deleteModifications(studyUuid));
     }
 
+    @PutMapping(value = "/studies/{studyUuid}/network-modification/lines/{lineId}/lockout")
```

Comment:
See review in network modification server

Reply:
Rest API modified